qemu/tests/boot_with_different_vectors.py

```diff
@@ -45,41 +49,168 @@ def boot_guest_with_vectors(vectors):
         vm.verify_alive()
         return vm
 
+    def get_MSI_queue_from_traceview_output(output):
+
```

Comment:
Could you add docstring for these functions, which will be more clear.thx

Reply:
Thanks, modified